taichi/aot/module_loader.h

```diff
@@ -94,6 +94,11 @@ class TI_DLL_EXPORT Module {
     TI_NOT_IMPLEMENTED;
   }
 
+  virtual const DeviceCapabilityConfig &get_required_caps() const {
+    static DeviceCapabilityConfig default_cfg;
```

Comment:
Correct me if I'm missing anything, `default_cfg` should respect the caps specified by users but I didn't see that logic in this PR?